### DISCUSSION TITLE
How to Make Calendar's resourceTimelineDay View Fill Full Width After Setting slotMinTime and slotMaxTime?

### DISCUSSION THREAD

**Discussion by nrdnisml** [GuavaCZ/calendar#54] [Q&A] [answered]
I'm using Calendar's resourceTimelineDay view, and I've set slotMinTime to 08:00:00 and slotMaxTime to 20:00:00. However, after applying these settings, the timeline view is cut to fit the length of the time range instead of stretching to fill the full width of the container.

Here's an example of what it looks like:
![image](https://github.com/user-attachments/assets/d9e84e88-1746-47c9-b7b5-70c571d57724)

How can I configure Calendar to make the timeline view span the full width of the container regardless of the time range?

**Comment by lukas-frey** [comment]
You can try playing with the CSS to see if you can fix it, the rendering is done via the underlaying vkurko/calendar package so nothing I can do about that. It behaves a little differently for each view.

**Reply by nrdnisml** [reply, marked as answer]
I managed to find a way by adding the slotWidth option in the getOptions() function to maintain the width of the event bar
```   
public function getOptions(): array
    {
        return [
            'slotMinTime' => '08:00:00',
            'slotMaxTime' => '21:00:00',
            'slotWidth' => '98',
        ];
    }
```
and adding the following code in custom css with the same px value  like this to preserve the timeline column's width
```
.ec-timeline .ec-line,
.ec-timeline .ec-time {
    width: 98px;
}
```
